server/db-pg/seedHelpers.js

```diff
@@ -19,50 +24,27 @@ const makeName = () => lorem.generateWords(2);
 
 const makeDescription = () => lorem.generateSentences(2);
 
-// TODO: refactor this
-const makeAddress = () => Math.floor(Math.random() * Math.floor(100)) + ' ' + lorem.generateWords(1) + ' ' + ['St.', 'Blvd.', 'Rd.'][Math.floor(Math.random() * Math.floor(3))];
+const randomInt = (min, max) => min + Math.floor(Math.random() * max);
+
+const makeAddress = () => `${randomInt(1, 300)} ${lorem.generateWords(1)} ${['St.', 'Blvd.', 'Rd.'][randomInt(0, 2)]}`;
 
-const makeDeliveryTime = () => Math.floor(Math.random() * Math.floor(60));
+const makeDeliveryTime = () => randomInt(1, 60);
 
 // TODO: fix this
 const makeLocation = () => [33.067149, -117.263955];
```

Comment:
You may already have plans for this, but I know that Faker can generate fake coords.